packages/account/src/Components/api-token/api-token-clipboard.tsx

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { useIsMounted } from '@deriv/shared';
 import { Button, Icon, Modal, Text, Popover } from '@deriv/components';
 import { localize } from '@deriv/translations';
 
-const WarningNoteBullet = ({ message }) => (
+type TApiTokenClipboard = {
+    scopes: string[];
+    text_copy: string;
+    info_message: string;
+    success_message: string;
+    popover_alignment?: import('Types').TPopoverAlignment;
```

Comment:
I'm not sure.. but I'd suggest to use common imports instead of dynamic. Just for better readability